certified-connectors/Airmeet/apiDefinition.swagger.json

```diff
@@ -1650,12 +1471,677 @@
+            }
+          }
+        ]
+      }
+    },
+    "/airmeet/{airmeetId}/session": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "Session added to your Airmeet",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "uuid": {
+                  "type": "string",
+                  "description": "Session ID"
+                }
+              }
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "airmeetId",
+            "in": "path",
+            "required": true,
+            "x-ms-url-encoding": "single",
+            "type": "string",
+            "x-ms-summary": "Airmeet ID",
+            "description": "Airmeet Event ID",
+            "x-ms-dynamic-values": {
+              "operationId": "GetAirmeets",
+              "value-path": "uid",
+              "value-collection": "data",
+              "value-title": "name",
+              "parameters": {
+                "allAirmeets": true,
+                "crmName": "MICROSOFT_DYNAMICS"
+              }
+            }
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "sessionTitle": {
+                  "type": "string",
+                  "description": "Session Title"
+                },
+                "sessionStartTime": {
+                  "type": "integer",
+                  "format": "int64",
+                  "description": "Session Start Time",
+                  "title": "Session Start Time"
+                },
+                "sessionDuration": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Session Duration",
+                  "title": "Session Duration",
+                  "default": 30
+                },
+                "sessionSummary": {
+                  "type": "string",
+                  "description": "Session Summary"
+                },
+                "hostEmail": {
+                  "type": "string",
+                  "description": "Host Email",
+                  "title": "Host Email"
+                },
+                "speakerEmails": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "Speaker Emails"
+                },
+                "cohostEmails": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "Cohost Emails"
+                },
+                "type": {
+                  "type": "string",
+                  "description": "type",
+                  "title": "Type",
+                  "default": "HOSTING",
+                  "enum": [
```

Comment:
@amit-vb The list of enum values has been added to the input field. Existing flows may use values outside of this list.
can you please confirm if the connector can handle values outside of the enum list?

Reply:
Hi @Amjed-Ayoub 
right now we have only internal users & so after the new version will send them a notification to use the latest versions.